### PR TITLE
Add affiliate instructions markdown

### PR DESCRIPTION
**Pull request by julo15**
Porting over android-affiliate-sdk.md from sewichi-agent, which the updated instructions to consume the SDK from our public Maven repo.

### CODE REVIEW

**Comment by nickneiman** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+### Adding the dependency to your project
+
+1. Add the following to your **root** `build.gradle` file:
+
+```
+allprojects {
+    repositories {
+        ...
+
+        maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
```
Comment (dgranahan) [inline README.md:14]: Should this be `https://github.com/placed/android-placed-sdk/master/repository`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+# Placed Affiliate SDK (Android) v1.20
+#### Last Updated: August 23, 2016
```
Comment (dgranahan) [inline README.md]: We can probably remove the date since it's now in source control.

File: README.md (hunk 1 above)
Comment (julo15) [reply README.md:14]: No, because that will point to the github webpage for that directory. We need the raw file directory.

File: README.md (hunk 2 above)
Comment (julo15) [reply README.md]: Good call

File: README.md (hunk 1 above)
Comment (dgranahan) [reply README.md:14]: Ah, fair enough.

**Comment by dgranahan** [comment]
Looks good to me 👍

**Pull request merged**